WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -164,6 +164,14 @@ public static int GetAnonymitySet(this Transaction me, int outputIndex, ICoinsVi
 				anonset += sanityCheckedEstimation;
 			}
 
+			// Factor in script reuse.
+			var output = me.Outputs[outputIndex];
+			foreach (var coin in allWalletCoins.FilterBy(x => x.ScriptPubKey == output.ScriptPubKey))
+			{
+				anonset = Math.Min(anonset, coin.AnonymitySet);
```

Comment:
is this enough? i think this should be counted as an additional intersection

also if we're checking for this, might also want to add sibling correction to this PR

Reply:
> is this enough? i think this should be counted as an additional intersection

Done.

Reply:
> also if we're checking for this, might also want to add sibling correction to this PR

Done.